deploy/service.yaml

```diff
@@ -56,7 +56,7 @@ spec:
       serviceAccountName: keptn-dynatrace-service
       containers:
         - name: dynatrace-service
-          image: keptncontrib/dynatrace-service:0.10.0
+          image: bacherfl/dynatrace-service:080
```

Comment:
Can you please set the `keptncontrib/dynatrace-service` repo

Reply:
thanks, I have missed that one. Jut fixed it